v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/FirestoreToBigQuery.java

```diff
@@ -68,7 +68,7 @@
       contactInformation = "https://cloud.google.com/support",
       hidden = true),
   @Template(
-      name = "Firestore_to_BigQuery_Flex",
+      name = "Firestore_to_BigQuery_Flex_Xlang",
```

Comment:
Do you think we should keep the naming convention consistent here? with just the `_Xlang` suffix